lemmatizer/README.md

```diff
@@ -161,3 +161,8 @@ TODO
 ##Citation
 
 Thank you for citing us if you use us in your work! http://cogcomp.cs.illinois.edu/page/software_view/illinois-lemmatizer
+
+```
+@inproceedings{
```

Comment:
If you use this code in your research, please provide the URL for this 
github repository in the relevant publications.